crates/routes/src/feeds.rs

```diff
@@ -54,18 +53,19 @@ impl Params {
   }
 }
 
-enum RequestType {
-  Community,
-  User,
-  Front,
-  Inbox,
-  Modlog,
-}
-
 pub fn config(cfg: &mut web::ServiceConfig) {
   cfg.service(
     web::scope("/feeds")
-      .route("/{type}/{name}.xml", web::get().to(get_feed))
+      .route("/u/{user_name}.xml", web::get().to(get_feed_user))
+      .route("/c/{community_name}.xml", web::get().to(get_feed_community))
+      .route("/front/{jwt}.xml", web::get().to(get_feed_front))
+      .route("/modlog/{jwt}.xml", web::get().to(get_feed_modlog))
+      .route("/notifications/{jwt}.xml", web::get().to(get_feed_notifs))
+      // Also redirect inbox to notifications. This should probably be deprecated tho.
+      .service(web::redirect(
+        "/inbox/{jwt}.xml",
+        "/notifications/{jwt}.xml",
+      ))
```

Comment:
In order to be able to use the redirect properly, I had to re-organize this a bit, and get rid of the `get_feed` fn.